docs/ux/menu-structure.md

```
@@ -0,0 +1,225 @@
+> - `apps/app_user/lib/src/routing/app_routes.dart`
+> - `apps/app_user/lib/src/ui/shell/user_scaffold.dart`
+> - `apps/app_partner/lib/src/routing/app_routes.dart`
+> - `apps/app_partner/lib/src/ui/shell/partner_scaffold.dart`
+
+---
+
+## 1. 유저 앱 (app_user)
+
+### 1.1 BottomNav (2탭)
+
+| # | 아이콘 | 라벨 | 경로 | 소스 파일 |
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)